recipes/gaia/build.sh

```diff
@@ -0,0 +1,16 @@
+
+# From the gaia readme:
+#
+# https://github.com/OpenDataAnalytics/gaia/blob/master/README.md#installation
+#
+# # Installation issues - Ubuntu:
+# You may run into cpl_port.h No such file or directory. Do the following:
+# sudo apt-get install libgdal-dev
+# export CPLUS_INCLUDE_PATH=/usr/include/gdal
+# export C_INCLUDE_PATH=/usr/include/gdal
+#
+export CPLUS_INCLUDE_PATH=${PREFIX}/include
+export C_INCLUDE_PATH=${PREFIX}/include
+
```

Comment:
Do you need this?

Reply:
When I installed gaia outside of conda, it failed until I provided these environment variables.  I didn't try the conda build without them though.  Does it hurt to have them, since they're documented in the gaia README?

Reply:
Why does a python package needs c++ variables? Even if that is the case, I would think the compiler already looks into ${PREFIX}/include. Can you try removing them? 

The installation outside conda uses pip and probably uses the system libgdal.

Reply:
Yes, I'll give it a try.